tc/sgx/common/crypto/sig_private_key.cpp

```diff
@@ -206,16 +206,13 @@ std::string pcrypto::sig::PrivateKey::Serialize() const {
 }  // pcrypto::sig::PrivateKey::Serialize
```

Comment:
ECDSA_do_sign and ECDSA_verify_sign functions do not compute the hash by itself. Generally the signing utility function accepts hashing algorithm, message in raw and the key to compute the signature output or to verify the signature.
Ref: https://github.com/openssl/openssl/blob/master/crypto/ec/ecdsa_sign.c#L14
https://github.com/openssl/openssl/blob/6f02932edba62186a6866e8c9f0f0714674f6bab/crypto/ec/ecdsa_vrf.c#L20

I guess the change has to be done in Python and not in C++.

Please add back comments as well, the name of the hashing algorithm which is always SHA256 for this sign/verify functionality.

Reply:
The parameter name message is confusing with new changes done in this PR. I'll change it to hashMessage